docs/en/observability/synthetics-analyze.asciidoc

```diff
@@ -0,0 +1,358 @@
+to help you quickly diagnose outages and other connectivity issues within your network.
+
+To access this page, go to *{observability}* -> *Synthetics* and make sure you're on the *Overview* tab.
+
+This overview includes a snapshot of the current status of all monitors, the number of errors that
+occurred over the last 6 hours, and the number of alerts over the last 12 hours.
+All monitors included your deployment will be listed below with information about the location,
+current status, and duration average.
+
+[NOTE]
+====
+When you use a single monitor configuration to create monitors in multiple locations, each location
+is listed as a separate monitor as they run as individual monitors and the status and duration average
+can vary by location.
+====
+
+[role="screenshot"]
+image::images/synthetics-monitor-page.png[Synthetics UI in {kib}]
+
+To get started with your analysis in the Overview tab, you can search for monitors or
+use the filter options including current status (up, down, or disabled),
+monitor type (for example, journey or HTTP), location, and more.
+
+Then click an individual monitor to see more details.
+
+// Integrate with other Observability apps??
+
+[discrete]
+[[synthetics-analyze-individual-monitors]]
+= All monitor types
+
+When you go to an individual monitor's page, you'll see much more detail about the monitor's
+performance over time. The details vary by monitor type, but for every monitor at the top of the
+page you'll see:
+
+* The monitor's *name* with a down arrow icon that you can use to quickly move between monitors.
+* The *location* of the monitor. If the same monitor configuration was used to create monitors in
+  multiple locations, you'll also see a down arrow icon that you can use to quickly move between
+  locations that use the same configuration.
+* The latest *status* and when the monitor was *last run*.
+* The *image:images/icons/beaker.svg[Beaker icon] Run test manually* button that allows you to run the test on
+  demand before the next scheduled run.
+* The *image:images/icons/pencil.svg[Pencil icon] Edit monitor* button that allows you to edit the monitor's
+  configuration.
+
+[role="screenshot"]
+image::images/synthetics-analyze-individual-monitor-header.png[Header at the top of the individual monitor page for all monitor types in the Synthetics UI]
+
+Each individual monitor's page has three tabs: Overview, History, and Errors.
+
+[discrete]
+[[synthetics-analyze-individual-monitors-overview]]
+== Overview
+
+The *Overview* tab has information about the monitor availability, duration, and any errors
+that have occurred since the monitor was created.
+The _Duration trends_ chart displays the timing for each check that was performed since the monitor was created.
+This visualization helps you to gain insights into how quickly requests resolve by the targeted endpoint
+and gives you a sense of how frequently a host or endpoint was down in your selected time span.
```

Comment:
Suggest removing "...in your selected time span..." as there is no time picker on the Overview (we're opinionated about the period shown)